src/Uno.UWP/Storage/StorageFile.cs

```diff
@@ -34,6 +34,18 @@ private StorageFile(Uri uri)
 			_fileUri = uri;
 		}
 
+		public async Task DeleteAsync()
```

Comment:
It's best if you invoke `DeleteAsync(CancellationToken.None)`. Also, to avoid the missing API issue you'll need to keep the `IAsyncAction` return type. Use `AsyncAction.FromTask()` to return one.

Reply:
When I tried to
IAsyncAction DeleteAsync() => DeleteAsync(CancellationToken.None);
it complains that Task is not convertible to IAsyncAction.
Yes, in Generated, we have IAsyncAction DeleteAsync(), but in API (and in DeleteAsync(CancellationToken ct)) we have Task.
So, maybe more appropriate is adding method to IgnoreSet?

Reply:
And, why build complains about missing
Windows.Foundation.IAsyncAction Windows.Storage.StorageFolder.DeleteAsync()
as in StorageFolder.cs, lines 217-221, method
public IAsyncAction DeleteAsync() =>
is defined?